src/main/java/net/pms/newgui/Wizard.java

```diff
@@ -108,6 +108,7 @@ public static void run(final PmsConfiguration configuration) {
 				configuration.setMinimized(false);
 			}
 
+/*
```

Comment:
Can we remove this commented chunk instead of commenting? We can always retrieve it from the git history if we need it again

Reply:
No problem